.chloggen/support-client-info-metadata.yaml

```diff
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: routingprocessor
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Enables processor to extract metadata from client.Info
+
+# One or more tracking issues related to the change
+issues: [20913]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
+  Enables processor to perform context based routing for payloads on the http server of otlp receiver
```

Comment:
Do we want to mention gRPC as well?

Reply:
@atoulme same as above, I could specify the grpc server but wanted to specify that it was specifically introducing new functionality for http